checkBitSize/README.md

```diff
@@ -0,0 +1,20 @@
+# checkBitSize
```

Comment:
Is this programm really required? DataCompressor reports the sizes of its base data types on startup

Reply:
As discussed we leave it there for good measure, but we'll use existing code form the DataCompressor.